src/View/Parsers/HtmlDiff.php

```diff
@@ -285,7 +286,7 @@ private static function cleanHTML(string $content, ?HTMLCleaner $cleaner = null)
     private static function getDiffer(): Differ
     {
         if (!HtmlDiff::$differ) {
-            HtmlDiff::$differ = new Differ();
+            HtmlDiff::$differ = new Differ(new DiffOnlyOutputBuilder());
```

Comment:
sebastian/diff had a major bump from 4 to 6 as it's a requirement of phpunit, the signature for the constructor changed